src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -275,7 +275,7 @@
             </ToolTip>
           </ComboBox.ToolTip>
           <ComboBox.ItemContainerStyle>
-            <Style TargetType="{x:Type ComboBoxItem}" BasedOn="{StaticResource ComboBoxItemStyle}">
+            <Style TargetType="{x:Type ComboBoxItem}" BasedOn="{StaticResource CommonControlsComboBoxItem}">
               <Setter Property="AutomationProperties.Name" Value="{Binding SourceName}"/>
```

Comment:
Why didn't I remove this?
Without the AutomationProperties.Name here, the Name ends up being the <Name: Name> which repeats twice unnecessarily. I'm not sure what is responsible for that behavior (guessing ToString), but I maintained our explicit setting of the property.

#### With Name set (as is):
![image](https://user-images.githubusercontent.com/49205731/152050366-82c7e71e-b39a-4880-a1d9-2d6eb88373a1.png)

#### Without setting the Name:

![image](https://user-images.githubusercontent.com/49205731/152050388-f0655f10-9103-4bb8-997e-cc82cfa9e8aa.png)